rotkehlchen/chain/manager.py

```diff
@@ -1410,6 +1420,21 @@ def _add_protocol_balances(self) -> None:
                     eth_balances[address].assets[asset_balance.asset] += asset_balance.balance
                     self.totals.assets[asset_balance.asset] += asset_balance.balance
 
+        liquity_module = self.get_module('liquity')
+        if liquity_module is not None:
+            liquity_balances = liquity_module.get_positions(
+                addresses=self.queried_addresses_for_module('liquity'),
+            )
+            for address, deposits in liquity_balances.items():
+                if 'trove' in deposits:
+                    collateral = deposits['trove'].collateral.balance  # type: ignore
+                    eth_balances[address].assets[A_ETH] += collateral
+                    self.totals.assets[A_ETH] += collateral
+                if 'stake' in deposits:
+                    deposited_lqty = deposits['stake'].staked.balance  # type: ignore
+                    eth_balances[address].assets[A_LQTY] += deposited_lqty
```

Comment:
I guess same question here. But probably since it's liquity makes sense to only stake LQTY

Reply:
Yes, you only stake LQTY